ui-tests/src/test/java/org/openmrs/reference/FormTest.java

```diff
@@ -109,10 +96,6 @@ public void deleteFormTest() throws Exception {
 	
 	@After
 	public void tearDown() throws Exception {
-		if (headerPage != null) {
-			headerPage.clickOnHomeIcon();
-			headerPage.waitForPageToLoad();
-			headerPage.logOut();
-		}
+	       homePage.go();
```

Comment:
We can as well here embrace a similar code indention which you have smartly addressed within `deletePatient()` method in the commit with id `79f836f`